### ISSUE TITLE
Migrating from iOS to e/OS - Signal locked me out

### ISSUE THREAD

**Issue by Maximilian-tek** [signalapp/Signal-Android#13332]
Hello there,
i am with my iPhone SE (2020) on the latest iOS version and the latest Signal Version (6.54). I am in the process to migrate all my data to a new Fairphone 5 with e/OS. Signal is the! last app to migrate,
everything else did work without any workaround (all my other apps). But here I am..

My doing was the following:
Install Signal on the FP5 (Aurora Store - 6.41.3) (latest)
Open Signal, type in my phone number and get the 1 time sms code, type that one in (everything fine)
type in my signal pin (thats the 100% correct one)
App Says I am locked out if my account remains 7 days inactive

Okay panic me! So going back to the iPhone and Re-Register my Device -> No Problem, typing in my Signal Pin and that works..

Re-Doing the migration process to the FP5 2 or 3 times over (same result as the 1. time) and now i am locked out for about 5 hours out of both phones, because I tried to many times.
So now my iPhone and Fairphone are locked out of Signal, and so is my Desktop PC.
I will Re-Register my Device in about 5 hours, and hopefully have then an iPhone with only Signal on it and my new Phone with the rest, but that is hopefully just temporary...

But what have I done wrong, where is the catch?
If any of you could help me, I am losing my mind here.

I tried my best with numerous signal posts, github issues, forum posts (but every answer was at least a few months old and not my usecase/not successfully answered).
If this post would be better suited in the official signal forums/ as a github issue, that will be my next step. Thank you all <3

Best suiting to my case would be maybe: https://github.com/signalapp/Signal-Android/issues/12835 but I would not be sure if this my exact case :/ Will try if I have no other option, but for now waiting for your input.

Sorry english is not my first language.

Disclaimer: Posted aswell on official Sub-Reddit

**Comment by lbrunkho** [comment]
I'm not too familiar with iOS Signal but if I was having issues registering, I would go to Signal on your iOS device > Settings > Account > Registration Lock and set it to  disable. Then wait an hour and try to transfer to your new phone. 

Also, if your e/OS install does not include google play services, I would install Signal directly from the website. https://signal.org/android/apk/

**Comment by Maximilian-tek** [comment]
Thanks for the help, I did disable the Registration lock (for now) and will try to migrate in about an hour. 
e/OS does come without the official Google Play Services, but has microG instead, nonetheless I think I read in the documentation, when the GooglePlay Services are unavailable that Signal fall backs on Web Socket? (thats not 100% correct, but IIRC). 
But yes, the apk download will also be on my list for the try in an hour. 
Will update : )

**Comment by Maximilian-tek** [comment]
Update:
Found the workaround, would not call that a solution:
Like you @lbrunkho mentionend I disabled the Registration Lock (which I do find a security concern), than I could set up the new Signal App (downloaded as apk) where I was prompted that my Signal Pin was wrong and I had to set a new one, and could use Signal on the Fairphone. All Chats are gone (I know that something is in the making for transfering, but sadly not right now) but I am still in all group chats, with the catch that the other parties have to write first, if not the groupchat is not accessable for me. 

I do have Signal on a Desktop PC and there my pinned Chats were visible (not the messages) and the groupchats could be found by searching the name. I do not know if this wipe of my chats has to do with the general migration or with my "resetting" my account due to the "not knowing" of my Signal Pin (which was 100% the correct one).

This issue can be closed, but is not fixed! I descriped the workaround, hopefully someone can use it : ) 

Thanks again to @lbrunkho for the help <3
